File: move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        count=0
        for i in nums:
            if i :
                nums[count]=i
                count+=1
        while(count<len(nums)):
            nums[count]=0
            count+=1
                

                

File: move-zeroes/move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int count =0;
        for(int n:nums){
            if (n!=0){
                nums[count]=n;
                count++;}}
            
        while(count<nums.length){
            nums[count]=0;
            count++;
                                 }
    }
}